src/modules/dashboards/types.ts

```diff
@@ -12,6 +12,14 @@ export type DashboardMetaData = {
   publicAccessKey: null | string;
 };
 
+type TypographySettings = {
```

Comment:
we have 

`export type Typography = {
  fontStyle: 'normal' | 'italic';
  fontWeight: 'normal' | 'bold' | 'lighter';
  fontSize: number;
  fontColor: string;
  fontFamily?: string;
  lineHeight?: string;
};` in `ui-core` that can be used here